package.json

```diff
@@ -8,9 +8,12 @@
     "compile": "npm run compile:css && npm run compile:js",
     "dev:css": "sass --load-path=node_modules --watch sass/etna.scss:templates/static/css/dist/etna.css",
     "dev:js": "webpack --config webpack.config.js --mode=development --watch",
+    "dev": "npm run dev:css & npm run dev:js",
     "test": "jest --coverage",
-    "lint": "prettier \"{scripts,sass}/**/*.{scss,js}\" --check",
-    "lint:fix": "prettier \"{scripts,sass}/**/*.{scss,js}\" --write"
+    "lint:js": "eslint 'scripts/**/*.js'",
+    "lint:js:fix": "eslint --fix 'scripts/**/*.js'",
+    "lint": "prettier --check '{scripts,sass}/**/*.{scss,js}' & stylelint 'sass/**/*.scss' & eslint 'scripts/**/*.js'",
+    "lint:fix": "prettier --write '{scripts,sass}/**/*.{scss,js}' & stylelint --fix 'sass/**/*.scss' & eslint --fix 'scripts/**/*.js'"
```

Comment:
Uses `&` rather than `&&` so that if there are errors from more than just one tool, we see all of them. It does mean the task doesn't exit cleanly and the user has to use `cntrl c`

Reply:
The reason I used `&&` was to ensure the second command only runs after the first has succeeded.

Just using one `&` will force the command to run in the background and can't be easily quit.

Using `&&` also ensured we we only linting what had already been nicely formatted and that we wouldn't get conflicts from the two processes both trying to write changes back to the file system.

Reply:
I'll change these ones. It does mean that linting may be slightly confusing, as it will only report css changes until you've fixed all of them, and only then report js ones - I think it might be slightly clearer if we split them out into two separate tasks.

Reply:
Switched running them consecutively - @ahosgood let me know if you're happy for me to split the linting into two tasks - js and css.

Reply:
As long as we end up with a single `lint` command that exits a non-zero status if either command fails (no running in the background) so we can use it in the CI, I'm happy 😊

Reply:
I think I'll leave it as one now with the double `&&` - it's not ideal but we probably want to keep it just one command for the CI pipeline.